KubeArmor/BPF/system_monitor.c

```diff
@@ -50,6 +50,9 @@
 #include "throttling.h"
 #include "common.h"
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 7, 0)
```

Comment:
some distributions like RHEL backports bpf-lsm support so it would be false negative in such cases.

Reply:
so how do we handle in such case?